core/app.js

```diff
@@ -296,12 +296,18 @@ app.use(_.get("/employees",function *(){
   var isAdmin = yield auth.isAdmin(this.session.user);
   var client = db();
   var allAdmins = yield client.query("SELECT netid, email FROM users WHERE admin = TRUE;");
+  var names = []
+  for(var x = 0; x < allAdmins.rows.length; x++){
+    var db_id = (allAdmins.rows[x]).netid;
+    var data = yield user.ldapInfo(db_id);
+    names.push(data[db_id].cn);
+  }
```

Comment:
`apply` is part of `Function.prototype` and so is a method on all functions (unless the prototype has been messed with). Basically, it turns an array of arguments into separate arguments:

```javascript
function a(){}
var b = [1,2,3];
a.apply(this, a);

// is the same as
a(b[0], b[1], b[2]);
```

The MDN has some pretty good documentation on it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply